tests/Doctrine/Tests/ORM/Functional/Ticket/GH7869Test.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Decorator\EntityManagerDecorator;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\UnitOfWork;
+use Doctrine\Tests\Mocks\ConnectionMock;
+use Doctrine\Tests\Mocks\DriverMock;
+use Doctrine\Tests\Mocks\EntityManagerMock;
+use Doctrine\Tests\OrmTestCase;
+
+/**
+ * @group GH7869
+ */
+class GH7869Test extends OrmTestCase
+{
+    public function testDQLDeferredEagerLoad()
```

Comment:
I forgot to rename test case 😞 Let me know if I Should create new PR with just this change